.github/workflows/e2e.yml

```diff
@@ -18,7 +18,7 @@ on:
 jobs:
   Run-wpe2e-TestCase:
     # The type of runner that the job will run on
-    name: Run rtsocial 
+    name: Playwright
```

Comment:
To make it consistent with other update it to Playwright/test

Reply:
Done